### PR TITLE
deploy: grab nodes for PGI license automatically

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

**Comment by matz-e** [comment]
Retest this please

File: deploy/deploy.lib [hunk 1]
```diff
@@ -315,7 +315,7 @@ configure_compilers() {
             makelocalrc ${PGI_DIR} -gcc ${GCC_DIR}/bin/gcc -gpp ${GCC_DIR}/bin/g++ -g77 ${GCC_DIR}/bin/gfortran -x -net
             #configure pgi network license
             template=$(find $PGI_DIR -name localrc* ! -iname "*.bak" | tail -n 1)
-            for node in bbpv1 bbpv2 bbptadm tds03 tds04 r2i3n0 r2i3n1 r2i3n2 r2i3n3 r2i3n4 r2i3n5 r2i3n6; do
+            for node in bbpv1 bbpv2 bbptadm tds03 tds04 $(squeue -h -o %N\ %j -u bbprelman|awk '/ BB5-[0-9]+$/ {print $1}'); do
```
Comment (pramodk) [inline deploy/deploy.lib]: you deleted UC2 nodes from the list :) 
similarly, we should grab all GPU nodes!

**Review by pramodk — CHANGES_REQUESTED** [review]

File: deploy/deploy.lib (hunk 1 above)
Comment (pramodk) [reply deploy/deploy.lib]: just to clarify : we need login nodes + tds nodes + uc2 nodes (p1+p2) + Jenkins nodes.

Comment (matz-e) [reply deploy/deploy.lib]: See update

**Pull request closed without merging**